### ISSUE TITLE
Project: Waze CCP Data Processor

### ISSUE THREAD

**Issue by schnuerle** [GovInTheOpen/Project-Portfolio#1]
August 2017 - Would take Waze CCP data feed and process it into your cloud provider for access, analysis, and visualization. https://www.waze.com/ccp

Even though there are over 500 Waze CCP partners, only a small percent use the data that Waze provides to govs.  It comes in 2 minute JSON file chunks, and it's up to govs to store and process the data themselves, and build the analytics tools and applications off the database.  

Louisville has processed the data to an internal Microsoft MS SQL database from June 2016 to February 2017, and we are connecting to it with SQL queries and Power BI, and ran an internal hackathon.

**Comment by schnuerle** [comment]
October 2017 - Louisville paid a local dev company around $2K to train us on AWS in general and how we could go about migrating this to the cloud.
 
Pitched this project to the Harvard Civic Analytics Network and got some interest.

**Comment by schnuerle** [comment]
November 2017 - We've published a first pass of our code and instructions to for cities to use on GitHub here:  https://github.com/LouisvilleMetro/WazeCCPProcessor

Right now it just takes your data feed URL and saves JSON to an S3 bucket.  But at least you are collecting the data for the first time.  Three cities have used it to do this: LA, Anchorage, Denver.

It also is built in Cloud Formation so it's easily deployable, updateable, and destroyable, and we have a future architecture diagram and detailed Project and Wiki area with tasks on the repo.

Other cities are interested in helping, including NYC, Boston, Pittsburgh, Philly, New Orleans, Chicago, San Francisco, and San Jose.

**Comment by schnuerle** [comment]
January 2018 - Ok it looks like this is a good candidate for a project, so let's make it official!

**Comment by schnuerle** [comment]
Reopening project.  Maybe it shouldn't be closed until it's completed.

**Comment by patwater** [comment]
Yes one thought would be to phase the project. Definitely seems like there could be future evolutions or opportunities to say integrate Waze data with work order information (lots of challenges to develop requisite standards across cities!) though also that the first phase is reaching a close. Been very cool to watch from the sidelines! Great model for inter-city (and other local government) data collaborations.
